src/consumeroffset/kafka_offset_collect.go

```diff
@@ -11,272 +10,42 @@ import (
 	"github.com/newrelic/infra-integrations-sdk/integration"
 	"github.com/newrelic/infra-integrations-sdk/log"
 	"github.com/newrelic/nri-kafka/src/args"
-	"github.com/newrelic/nri-kafka/src/connection"
 )
 
-// getConsumerOffsets collects consumer offsets from Kafka brokers rather than Zookeeper
-func getConsumerOffsets(groupName string, topicPartitions TopicPartitions, client connection.Client) (groupOffsets, error) {
```

Comment:
All this deleted part is functions we don't need anymore because they were called when the deprecated flag was in place.